<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventOrTodoDesktop</class>
 <widget class="QWidget" name="EventOrTodoDesktop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>308</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="@info:tooltip">Set the title</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Sets the Title of this event or to-do.</string>
     </property>
     <property name="text">
      <string extracomment="@label">T&amp;itle:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>mSummaryEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="KLineEdit" name="mSummaryEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string extracomment="@info:tooltip">Set the title</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Sets the Title of this event or to-do.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string extracomment="@label">&amp;Location:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>mLocationEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="KLineEdit" name="mLocationEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string extracomment="@info:tooltip">Set the location</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Sets where the event or to-do will take place.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mStartLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Start:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>mStartDateEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="mStartCheck">
     <property name="whatsThis">
      <string>Sets the start date for this to-do</string>
     </property>
     <property name="text">
      <string extracomment="@option:check to-do start datetime"/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDateEdit" name="mStartDateEdit">
     <property name="displayFormat">
      <string>MM.dd.yyyy</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QTimeEdit" name="mStartTimeEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="KPIM::KTimeZoneComboBox" name="mTimeZoneComboStart">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Select the timezone for this event. It will also affect recurrences</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="mEndLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;End:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>mEndDateEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="mEndCheck">
     <property name="whatsThis">
      <string>Sets the due date for this to-do.</string>
     </property>
     <property name="text">
      <string extracomment="to-do due datetime"/>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDateEdit" name="mEndDateEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>MM.dd.yyyy</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QTimeEdit" name="mEndTimeEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="KPIM::KTimeZoneComboBox" name="mTimeZoneComboEnd">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Select the timezone for this event. It will also affect recurrences</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QCheckBox" name="mWholeDayCheck">
     <property name="whatsThis">
      <string>Set if this to-do's start and due dates have times associated with them.</string>
     </property>
     <property name="text">
      <string>Whole day</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QWidget" name="mCompletionPriorityWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSlider" name="mCompletionSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mCompletedLabel">
        <property name="text">
         <string>completed</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Priority:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mPriorityCombo">
        <property name="currentIndex">
         <number>5</number>
        </property>
        <item>
         <property name="text">
          <string>unspecified</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 (highest)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 (medium)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9 (lowest)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mTaskLabel">
     <property name="text">
      <string>Task:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QCheckBox" name="mFreeBusyCheck">
     <property name="text">
      <string>Blocks me for other events</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPIM::KTimeZoneComboBox</class>
   <extends>KComboBox</extends>
   <header>ktimezonecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
